lib/geo-schema-types.js

```diff
@@ -157,6 +159,18 @@ class SchemaTypeGeoJSON extends SchemaType {
 		return result;
 	}
 
+	toJSONSchema(subschema, schema) {
+		_.assignIn(schema.jsonSchemaDefinitions, GeojsonJsonSchema.definitions);
+
+		let jsonSchema = GeojsonJsonSchema.schema;
+
+		if (subschema.description) {
+			jsonSchema = objtools.deepCopy(jsonSchema);
```

Comment:
You should always do the deep copy. Otherwise the result of this function will only conditionally have side effects.